src/commands/kv/namespace/site.rs

```diff
@@ -9,23 +8,23 @@ use crate::settings::global_user::GlobalUser;
 use crate::settings::target::Target;
 use crate::terminal::message;
 
-pub const MAX_NAMESPACES_PER_PAGE: u32 = 100;
-pub const PAGE_NUMBER: u32 = 1;
-
 pub fn site(
     target: &Target,
     user: &GlobalUser,
     preview: bool,
 ) -> Result<WorkersKvNamespace, failure::Error> {
     kv::validate_target(target)?;
-    let client = kv::api_client(user.to_owned())?;
 
     let title = if preview {
         format!("__{}-{}", target.name, "workers_sites_assets_preview")
     } else {
         format!("__{}-{}", target.name, "workers_sites_assets")
     };
 
+    // We call CreateNamespace here directly because we want raw access to the
+    // response codes; the implementations in commands::kv::namespace cannot surface
+    // the raw response
```

Comment:
should we implement the same `::call_api` pattern here as you have for the `namespace::list` module below?

Reply:
I wanted to, but ran into some challenges--check out the message above. I think I can do this by wrapping the `ApiError` in a `failure::Error` and extracting it via match. I'll give that a shot!